e2e-tests/path-prefix/package.json

```
@@ -20,22 +20,27 @@
   ],
   "license": "MIT",
   "scripts": {
-    "prebuild": "rm -rf assets && mkdir -p assets/blog",
+    "prebuild": "del-cli -f assets && make-dir assets/blog",
     "build": "gatsby build --prefix-paths",
-    "postbuild": "cp -r public/. assets/blog",
+    "postbuild": "cpy --cwd=./public --parents **/* ../assets/blog",
```

Ok, this one doesn't seem to actually copy files from public to assets/blog (on my machine)

---

```suggestion
    "postbuild": "cpy --cwd=./public --parents '**/*' '../assets/blog'",
```
this seems to fix it on my machine